lib/rules/no-extra-semicolons/index.js

```diff
@@ -53,6 +53,10 @@ const rule = function (actual) {
     }
 
     root.walk(node => {
+      if (node.mixin) {
```

Comment:
> Would that be of interest or is it too specific since it's a non-standard syntax thing?

Too specific I'm afraid. Rules make use of the `isStandardSyntax*` utils to make sure only _whitelisted_ constructs are dealt with within the rule. There are too many non-standard syntaxes to add special cases to individual rules, even if it's to explicitly ignore a piece of non-standard syntax. There are a lot of rules to maintain and having this consistent behaviour has benefited us thus far.

As such, I suggest we structure that conditional in terms of standard constuct and standard syntax utils i.e.:

```js
     if (
        node.type === "rule"
        && !isCustomPropertySet(node)
        && !isStandardSyntaxRule(node)
      ) {
        return
      }
```